invenio_rdm_records/config.py

```diff
@@ -174,6 +171,8 @@ def _(x):
 THEME_SITEURL = "http://127.0.0.1:5000"
 
 #: DataCite DOI credentials
-# RDM_DATACITE_CLIENT_USERNAME = ""
-# RDM_DATACITE_CLIENT_PASSWORD = ""
-# RDM_DATACITE_CLIENT_PREFIX = ""
+RDM_RECORDS_DOI_DATACITE_ENABLED = True
+RDM_RECORDS_DOI_DATACITE_USERNAME = ""
+RDM_RECORDS_DOI_DATACITE_PASSWORD = ""
+RDM_RECORDS_DOI_DATACITE_PREFIX = "10.1234"
```

Comment:
comment: this is the old test prefix, it will not enable datacite credentials since user and psw are still missing, but is more friendly to have this...